gem/styles/gem-springster/layout/_l-header.scss

```diff
@@ -3,7 +3,7 @@ html, body {
 }
 .header {
   overflow: auto;
-  padding: 10px 10px 5px;
+  padding: 10px 10px 0 5px;
```

Comment:
is this correct? bottom 0px and then right 10px and left 5px? initially it was set to have top, left and right 10px and 5px bottom